src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -24810,7 +24810,9 @@ public struct ValueTuple<T1, T2>
             var libWithVTRef = libWithVT.EmitToImageReference();
 
             var comp = CSharpCompilation.Create("test", references: new[] { libWithVTRef, corlibWithVTRef });
-            Assert.True(comp.GetWellKnownType(WellKnownType.System_ValueTuple_T2).IsErrorType());
+            var found = comp.GetWellKnownType(WellKnownType.System_ValueTuple_T2);
+            Assert.False(found.IsErrorType());
```

Comment:
>Assert.False(found.IsErrorType()); [](start = 12, length = 34)

Do we have a test covering an ambiguity that doesn't involve core library?

Reply:
"attribute in two libraries" scenario in `GetWellKnownType` test

---
In reply to: [585742024](https://github.com/dotnet/roslyn/pull/51571#discussion_r585742024) [](ancestors = 585742024)

Reply:
> "attribute in two libraries" scenario in GetWellKnownType test

Are you referring to PickAmbiguousAttributeFromCorlib unit-test? It doesn't look like it tests ValueTuple type.

---
In reply to: [585931231](https://github.com/dotnet/roslyn/pull/51571#discussion_r585931231) [](ancestors = 585931231,585742024)